ci/check-protobuf-stability.sh

```diff
@@ -70,6 +70,10 @@ check_protos() {
   check_non_lf_protos
 }
 
+version_lte() {
+  [ "$1" = "$(echo -e "$1\n$2" | sort -V | head -n 1)" ]
```

Comment:
Can we use `semver compare` instead? We already have the semver tool in dev-env.

Reply:
done